js/modules/k6/browser/common/page.go

```diff
@@ -546,10 +543,7 @@ func (p *Page) consoleMsgFromConsoleEvent(e *runtime.EventConsoleAPICalled) (*Co
 func (p *Page) closeWorker(sessionID target.SessionID) {
 	p.logger.Debugf("Page:closeWorker", "sid:%v", sessionID)
 
-	if worker, ok := p.workers[sessionID]; ok {
-		worker.didClose()
```

Comment:
Maybe the method name `closeWorker` also should be re-visited, since now, it's more about cleaning ? :thinking:

Reply:
Renamed it to `removeWorker` in 38d7d79a756fdea45db8ed2a89e56d0007eab988